scala/src/main/org/apache/spark/streaming/api/csharp/CSharpDStream.scala

```diff
@@ -224,20 +224,33 @@ class CSharpStateDStream(
                           serializationMode2: String)
   extends DStream[Array[Byte]](parent.ssc) {
 
-  super.persist(StorageLevel.MEMORY_ONLY)
+  val localCheckpointEnabled = parent.ssc.sc.conf.getBoolean("spark.mobius.localCheckpoint.enabled", false)
```

Comment:
let's add "streaming" to the config settings - like spark.mobius.streaming.localCheckpoint.enabled and spark.mobius.streaming.localCheckpoint.replicas